index.js

```diff
@@ -98,22 +98,31 @@ MarketplaceClient.prototype.validatePackage = function(packagePath) {
   return promise;
 };
 
-MarketplaceClient.prototype.publish = function(validatedManifestId) {
+MarketplaceClient.prototype.publish = function(validationId, type) {
+  if (type === null) {
```

Comment:
@freaktechnik this should also check if `type === undefined`, no? Idea here is if a user is just validating a manifest URL, they can still call `publish(validationID)` leaving the type param out altogether, yes?